bot.js

```diff
@@ -1,282 +1,272 @@
-const Discord = require('discord.js');
+const Discord = require("discord.js");
 // const ytdl = require('ydtl-core');
-const snoowrap = require('snoowrap');
-const config = require('./config/config.json')[process.env.NODE_ENV || 'dev'];
-const secrets = require('./config/secrets.json')[process.env.NODE_ENV || 'dev'];
+const snoowrap = require("snoowrap");
+const config = require("./config/config.json")[process.env.NODE_ENV || "dev"];
+const secrets = require("./config/secrets.json")[process.env.NODE_ENV || "dev"];
 const client = new Discord.Client();
 const maxLinks = 10;
 
-var CronJob = require('cron').CronJob;
+var CronJob = require("cron").CronJob;
 var redditLinks = [];
 var messageHistory = [];
 var archiveHistory = [];
 
-
 // When the client is ready, run this code
 // This event will only trigger one time after logging in
-client.once('ready', () => {
-	console.log('Ready!');
+client.once("ready", () => {
+  console.log("Ready!");
 });
 
-
 // Login to Discord with your app's token
-client.login(secrets.discord.token)
-	.then(() => {
-		console.log('Logged in!');
-	});
+client.login(secrets.discord.token).then(() => {
+  console.log("Logged in!");
+});
 
 // Setup oAuth for Reddit Api calls
 const r = new snoowrap({
-	userAgent: 'A Discord Bot that ocassionally posts content links from given subreddits to Discord',
-	clientId:     secrets.reddit.clientId,
-	clientSecret: secrets.reddit.clientSecret,
-	username:     secrets.reddit.username,
-	password:     secrets.reddit.password,
+  userAgent: "A Discord Bot that ocassionally posts content links from given subreddits to Discord",
+  clientId: secrets.reddit.clientId,
+  clientSecret: secrets.reddit.clientSecret,
+  username: secrets.reddit.username,
+  password: secrets.reddit.password,
 });
 
 // Job runs at 16:20:00 every day
 // Gets all the image links of the day and puts them in a random order
-new CronJob('00 20 16 * * *', () => {
-	dailyAnimePost();
-}, null, true, 'America/Los_Angeles');
+new CronJob(
+  "00 20 16 * * *",
+  () => {
+    dailyAnimePost(client.channels.get(config.discord.animeChannelID));
```

Comment:
This is the only "real" change. `dailyAnimePost` now accepts a channel param that all activity will be sent to. On messages, she will respond to the message's channel. The CRON job has to pass in the channel from the config.